### DISCUSSION TITLE
Gatsby AirTable Plugin

### DISCUSSION THREAD

**Discussion by ZeldOcarina** [gatsby-uc/plugins#367] [Ideas]
Hi wonderful community,

Unfortunately the [gatsby-source-airtable](https://github.com/jbolda/gatsby-source-airtable) community plugin is completely unmantained with the latest commit having been well over seven months ago.

There's multiple issues opened directly from AirTable engineers and it's becoming deprecated quite soon:

[484](https://github.com/jbolda/gatsby-source-airtable/issues/484), [483](https://github.com/jbolda/gatsby-source-airtable/issues/483).

There is some community alternatives but I think AirTable is in such a wide use now that it would deserve to enter gatsby-uc.

The current plugin is well written, it might be cloned and updated and it might not bee too much of a pain to do.

Since AirTable is thriving these days and it works amazingly well as a CRM for websites, I think it could be cool to give it some consideration:

https://www.mksguide.com/airtable-user-and-company-stats/

I am willing to take care of it, with some help from the community, it would be my first real open source project.

What do you think?

**Comment by moonmeister** [comment]
Hey! Great suggestion! Our goal is to move plugins usually not just fork them, I will reach out to the current maintainer and see if he is interested and willing to move the plugin into the UC. If that doesn't happen we can discuss alternatives.

**Comment by ZeldOcarina** [comment]
That would be amazing!! Let me know how I can help! :)

**Comment by moonmeister** [comment]
I reached out to Jacob on Discord, I realized I've interacted with him a couple times over there in the Party Corgy community. If he gives the go ahead I'll have him transfer the repo and such, then you can help with the work of integrating the plugin into the GUC monorepo.

**Comment by ZeldOcarina** [comment]
Ok perfect I look forward for that!

**Comment by moonmeister** [comment]
Moving forward to bring the plugin into the repo. https://github.com/gatsby-uc/plugins/issues/368

@ZeldOcarina, @jbolda has offered to do the PR to bring the plugin into the repo. Feel free to follow along and provide any feedback on discussions during that process. Once done you'll be able to open PRs to add features, etc.

**Reply by ZeldOcarina** [reply]
Amazing @moonmeister @jbolda thank you so much for your incredible contribution to open source! You guys are amazing!!

**Reply by jbolda** [reply]
Shooting to make some time tomorrow (Friday) to make this PR.
